code/datums/components/uplink.dm

```diff
@@ -49,7 +49,7 @@ GLOBAL_LIST_EMPTY(uplinks)
 		RegisterSignal(parent, COMSIG_PEN_ROTATED, .proc/pen_rotation)
 
 	GLOB.uplinks += src
-	uplink_items = get_uplink_items(gamemode, TRUE, allow_restricted)
+	uplink_items = get_uplink_items(_gamemode, TRUE, allow_restricted)
```

Comment:
this is just a bugfix, `gamemode` refers to the uplink gamemode var and it's null here always, it's only set later.